pkg/config/config.go

```diff
@@ -268,3 +269,14 @@ func (config *ConfigurationBuilder) AccountSvc() accountiface.Servicer {
 
 	return accountSvc
 }
+
+// LeaseSvc returns the lease Service for you
+func (config *ConfigurationBuilder) LeaseSvc() leaseiface.Servicer {
```

Comment:
I'm working to rename this to AccountService so we should probably change this to LeaseService.

Reply:
renamed